src/ui/components/buttons/SubcategoryButton/SubcategoryButtonListWrapper.native.test.tsx

```diff
@@ -18,7 +18,7 @@ jest.mock('libs/firebase/analytics/analytics')
 jest.mock('features/navigation/TabBar/routes')
 
 describe('<SubcategoryButtonListWrapper/>', () => {
-  it('should display "Films à l’affiche" instead of "Séances de cinéma" if offerCategory is "Cinema"', async () => {
+  it('should display "Films à l’affiche" instead of "Séances de cinéma" when offerCategory is "Cinema"', async () => {
```

Comment:
Je pense qu'on peut fusionner ce test et celui d'après, sinon c'est comme si on garder l'historique des changements dans les tests.
Juste un test avec `should display "Films à l’affiche" when offerCategory is "Cinema"` devrait suffire.

Reply:
Perso je n'aurais pas gardé ce test quel, on ne test plus du code (vu que tu as supprimé la condition) mais un comportement que l'on peut considérer comme étant legacy donc pourquoi garder ce test ? On pourrait avoir un test générique sur le fait d'afficher le bon label pour une catégorie donnée

Reply:
Et si pas envie de généraliser le test, à minima, avoir un test comme Thomas a proposé `should display "Films à l’affiche" when offerCategory is "Cinema"`